CONTRIBUTING.md

```diff
@@ -114,6 +114,10 @@ page on GitHub** -- though we still recommend you do not make changes there
 directly there but rather in a local clone, as that will make it easier for you
 to iterate during the PR review process.
 
+If you create a new page make sure to index it appropriately in the correct
+overview page. If you delete a page, make sure to set up a redirect in the
```

Comment:
question: how do we know what the "correct overview page" is? where do we find them?

Reply:
Updated the language to make it clearer.